brigadier/projects/models.py

```diff
@@ -0,0 +1,108 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from employees.models import Employee
+
+NEW = 'new'
+COMPLETED = 'completed'
+PROCESSED = 'processed'
+STATUSES = [
+    (NEW, _('New')),
+    (COMPLETED, _('Finished')),
+    (PROCESSED, _('Processed')),
+]
+STATUSES_DICT = dict(STATUSES)
+
+
+def get_full_name_of_employee(employee):
```

Comment:
this is not necessary

Reply:
I couldn't find a way for native searching in the list of tuples

Reply:
this marker is for get_full_name_of_employee()
this function is not needed